tests/unit/services/apigateway/test_mock_integration.py

```diff
@@ -61,6 +61,9 @@ def test_custom_parser(self, create_default_context):
             "{statusCode: 200, foo}: [ba r]}",
             "{statusCode: 200, foo'}: [ba r]}",
             "{statusCode: 200, }foo: [ba r]}",
+            "{statusCode: 200, }foo: ''}",
+            '{statusCode: 200, " ": " "}',
+            '{statusCode: 200, "": ""}',
```

Comment:
we could also add a few cases with `statusCode` in single and double quotes, and maybe a few invalid with non closing quotes or something similar?

Reply:
Good thinking! that is indeed the most important objective :rofl: